doc/storing-data.md

```diff
@@ -50,11 +61,20 @@ multiple_choice :question_2? do
     'q2-precalculated-answer'
   end
 
+  on_response do |response|
+    # response             => 'q2_option'
+    # responses            => ['q1_option']
+    # q1_calculated_answer => 'q1-calculated-answer'
+
```

Comment:
I think this `on_response` block would also have access to the `q2_precalculated_answer`, wouldn't it?